rep-2001.rst

```diff
@@ -773,6 +775,33 @@ Desktop Full
       packages: [perception, simulation, ros_ign_gazebo_demos]
 
 
+ROS Distribution Independent (September 2022 - Ongoing)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+These variants are not released as ROS packages within each distribution as the others are.
+Instead they are made available as independent packages outside of any ROS distribution.
+
+Build Essential
+"""""""""""""""
+
+::
+
+ - ros-build-essential:
+ - ros-build-essential:
+      packages: [cmake, git, python3-setuptools, wget]
```

Comment:
`wget` isn't needed for package builds and should be in ros-dev-tools instead.

```suggestion
      packages: [cmake, git, python3-setuptools]
```

Reply:
@cottsay should we make Python 3 itself explicitly required?

```suggestion
      packages: [cmake, git, python3, python3-setuptools]
```

Reply:
> should we make Python 3 itself explicitly required?

It wouldn't hurt to call it out explicitly, especially in an increasingly possible setuptools-free future.

Reply:
Add python3: [8a2e8ae](https://github.com/ros-infrastructure/rep/pull/355/commits/8a2e8ae56c9a1bc411d1bd341cae67497934a324)

Move wget: [91f54b3](https://github.com/ros-infrastructure/rep/pull/355/commits/91f54b31cd23e368829914d4cbc9d6a696f4f0b5)